### DISCUSSION TITLE
TPU, the retract test result is bad by using the retract tower result

### DISCUSSION THREAD

**Discussion by mhsieh66** [5axes/Calibration-Shapes#112] [General]
I have PRILINE 98A TPU filament and ran the Temperature Tower and the Retract Tower calibration, it's 205ºC, 1.75mm, and 10mm/s, I applied this result to do the Retact Test, but the result is bad, it doesn't clean like what I got from Retract Tower, it generates a lot of strings. 

Is there something I missed? It doesn't have any different results if using other types of filament.

**Comment by 5axes** [comment]
Tpu is difficult to print especialy if you have a bowden extruder. It's not really possible to supress stringing.

**Comment by mhsieh66** [comment]
Got it. My machine is CR10S Pro v2, and I modified it to direct drive. That's very interesting that the result is very clean if doing the Retract Tower test, but the result is trouble for the Retract Test.

**Comment by 5axes** [comment]
That's normal have a look to the wiki page  (https://github.com/5axes/Calibration-Shapes/wiki/Retract-Test-part)  . The Retract Tower is supposed to be an "easy" test  part to find the standard rectract parameter. With the Retract Test part Some parameter like "Minimum Layer Time" can have an influence on the stringing result.
